### PR TITLE
Review and refactor

### PR DESCRIPTION
**Pull request by reid-andrew**
Minor changes -
- Standardized model test naming conventions
- Reformat HTML on Merchant Index view so it renders correctly.

### CODE REVIEW

File: app/views/merchants/_merchants_index.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <p align="center"><%= link_to "New Merchant", "/merchants/new" %></p>
 <section class = "grid-container">
   <% @merchants.each do |merchant|%>
-    <section class = "grid-item" id = <%= "merchant_#{merchant.id}"%>>
+    <section id = <%= "merchant_#{merchant.id}"%>>
```
Comment (reid-andrew) [inline app/views/merchants/_merchants_index.html.erb:5]: This class was pulling in CSS that made this render in an unappealing way.

**Review by mlynch5187 — APPROVED** [review]
Looks good, and appear to have covered every test's syntax.

File: app/views/merchants/_merchants_index.html.erb (hunk 1 above)
Comment (javier-aguilar) [reply app/views/merchants/_merchants_index.html.erb:5]: lol. It really did

**Review by javier-aguilar — APPROVED** [review]
Looks good. The only thing is that the order status comments on spec tests will come back ex: # Cancelled Orders, however, this is not a major issue.

**Pull request merged**